lib/clearance/configuration.rb

```diff
@@ -133,6 +133,16 @@ def user_id_parameter
     def routes_enabled?
       @routes
     end
+
+    # Reloads the clearance user model class.
+    # This is called from the Clearance engine to reload the configured
+    # user class during each request while in development mode.
+    # @private
+    def reload_user_model
+      if @user_model.present?
+        @user_model = @user_model.to_s.constantize
```

Comment:
```
 pry(main)> ::User.to_s.constantize
=> User (call 'User.connection' to establish a connection)
::User
=> User (call 'User.connection' to establish a connection)
```

since these return the same value, do we need to `to_s` and `constantize`?